envoy/ssl/handshaker.h

```diff
@@ -177,27 +176,26 @@ class HandshakerFactory : public Config::TypedFactory {
 
 struct SelectionResult {
   enum class SelectionStatus {
-    // Continue the TLS handshake.
-    Continue,
-    // Block the TLS handshake.
-    Stop,
-    // Abort the TLS handshake.
-    AbortHandshake,
+    // TLS handshake should continue.
```

Comment:
It seems like these comments are explain how the enum should be used, not what they actually mean. So how about:
```
// A certificate was successfully selected.
Success,
// Certificate selection will complete asynchronously later.
Pending,
// Certificate selection failed.
Failed,
```